src/test/common/page/page.js

```diff
@@ -198,6 +200,35 @@ class Page {
     const hiddenErrorText = await hiddenError.getText()
     return fullErrorText.replace(hiddenErrorText, '').trim()
   }
+
+  async getCurrentSessionId () {
+    const cookies = await this.getCookies()
+    const sessionCookie = cookies.find(this.getCookieByName('htbhf.sid'))
+    return (typeof sessionCookie !== 'undefined') ? sessionCookie.value : null
+  }
+
+  async getLangCookie () {
+    const cookies = await this.getCookies()
```

Comment:
Should this line be in the `getCookieByName` method? This would stop it having to be defined in every method that aliases `getCookieByName` (`getCurrentSessionId` and `getLangCookie`).

Reply:
Yep, good spot, I'll change.